app/view-post/[postId].tsx

```diff
@@ -0,0 +1,86 @@
+import {
+    ActivityIndicator,
+    TouchableOpacity,
+    View,
+    FlatList,
+} from "react-native";
+import ItemCard from "../../components/view-post/item-card";
+import { useLocalSearchParams, useRouter } from "expo-router";
+import { PostContext } from "@/contexts/postContext";
+import useGetPostItems from "@/hooks/useGetPostItems";
+import useGetPost from "@/hooks/useGetPost";
+import PostHeader from "@/components/post-header";
+import { useContext } from "react";
+import { Divider, Text } from "@ui-kitten/components";
+
+export default function ViewPostPage() {
+    const router = useRouter();
+    const { postItems, post } = useContext(PostContext);
+    const { postId } = useLocalSearchParams();
+
+    const postItemsIsLoading = useGetPostItems(postId as string);
+    const postIsLoading = useGetPost(postId as string);
```

Comment:
can you change to 

```tsx
const { isLoading: postItemsIsLoading } = useGetPostItems(postId as string);
```

because kinda weird nga ang i return ng useGetPostItems is loading state lang